protocols/direct-funding.go

```diff
@@ -148,7 +148,10 @@ func (s DirectFundingObjectiveState) Update(event ObjectiveEvent) (Objective, er
 
 		err := updated.applySignature(sig, turnNum)
 		if err != nil {
-			return s, err
+			// If there was an error applying the signature, log it and swallow it
+			// This is a conscious choice (to ignore signatures we don't expect)
+			// Examples include faulty signatures, signatures by non-participants, signatures on unexpected states, etc
+			fmt.Println(err)
```

Comment:
I think this is appropriate, but it raises the question of logging, which I've given about zero thought to.

Reply:
#70